### PR TITLE
Add baseline model

### PR DESCRIPTION
**Pull request by huizhuansam**
Implemented a very basic CNN model, mostly by following the tutorial at https://www.tensorflow.org/tutorials/images/classification

The training data used has not been preprocessed or cleaned.

Approximately 60% accuracy on test data. More details are in the notebook's output.

### CODE REVIEW

**Review by deeyonn — APPROVED** [review]
LGTM. I noticed you're using tensorflow. I've had some problems with it because versions after `2.10` doesn't support GPU for windows. Something we can work out while we progress. Currently I'm using PyTorch to preprocess the data.
